js/yaaw.js

```diff
@@ -729,7 +745,8 @@ var YAAW = (function() {
 
     setting: {
       init: function() {
-        this.jsonrpc_path = $.Storage.get("jsonrpc_path") || location.protocol+"//"+(location.host.split(":")[0]||"localhost")+":6800"+"/jsonrpc";
+        this.jsonrpc_path = $.Storage.get("jsonrpc_path") ||"http://localhost:6800/jsonrpc";
```

Comment:
Suppose someone using YAAW on NAS. Try host address would make it no need to submit the jsonrpc path.